UI/MapboxNavigationUI/MGLMapView.swift

```diff
@@ -0,0 +1,96 @@
+import Foundation
+import Mapbox
+import MapboxDirections
+import MapboxGeocoder
+
+let sourceIdentifier = "routeSourceID"
+let routeLayerIdentifier = "routeLayerIdentifier"
+let routeLayerCasingIdentifier = "routeLayerCasingIdentifier"
```

Comment:
The string values themselves don’t need to contain “ID” or “identifier” in them.